config/prod/studies/mobile-toolbox-pmbfzc.yaml

```diff
@@ -0,0 +1,30 @@
+template_path: study-pipeline-infra.j2
+stack_name: '{{ stack_group_config.namespace }}-pmbfzc'
+dependencies:
+  - prod/glue-job-role.yaml
+  - prod/s3-intermediate-bucket.yaml
+  - prod/s3-parquet-bucket.yaml
+  - prod/glue-classifier-array-of-records.yaml
+  - prod/glue-job-S3ToJsonS3.yaml
+parameters:
+  Namespace: {{ stack_group_config.namespace }}
+  AppName: mobile-toolbox
+  StudyName: pmbfzc
```

Comment:
Is this an acronym?

Reply:
It's randomly generated by Bridge as far as I can tell.